### PR TITLE
ChangedItemEventArgs : IItemEvent

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Review by Thundermaker300 — APPROVED** [review]

**Comment by VALERA771** [comment]
Why not move NewItem to obsolete?

**Comment by joker-119** [comment]
It should be marked obsolete for the sake of removing it later.

**Comment by louis1706** [comment]
> It should be marked obsolete for the sake of removing it later.
i thinks it's better to keep the one with NewItem even if it's the same than Item to keep than it's more intuitive

**Review by IRacle1 — APPROVED** [review]

**Comment by joker-119** [comment]
I respectfully disagree, there's no reason for that kind of redundancy, not to mention if you use ev.Item, in this way, it'll be calling two property get methods instead of one, meaning it's objectively slower.

File: Exiled.Events/EventArgs/Player/ChangedItemEventArgs.cs [hunk 1]
```diff
@@ -43,11 +45,17 @@ public ChangedItemEventArgs(Player player, ItemBase oldItem)
         /// <summary>
         ///     Gets the new item.
         /// </summary>
-        public Item NewItem { get; }
+        [Obsolete("Use ev.Item instead of this")]
+        public Item NewItem => NewItem;
```
Comment (VALERA771) [inline Exiled.Events/EventArgs/Player/ChangedItemEventArgs.cs:49]:
```suggestion
        public Item NewItem => Item;
```

**Review by VALERA771 — CHANGES_REQUESTED** [review]

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**